Graphs/Table/index.js

```diff
@@ -731,8 +731,8 @@ const TableGraph = (props) => {
                                                     setCurrentStartIndex(props.startIndex);
                                                     onRowsRendered(props);
                                                 }}
-                                                width={graphWidth}
-                                                height={height}
+                                                width={graphWidth || 500}   
```

Comment:
Why do we need this change? This is going to affect all Tables.
We should not modify source code in test cases PR.

Reply:
Changes done.